app/config/env.php

```diff
@@ -82,3 +82,20 @@
     $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__ . '/cache_pool'));
     $loader->load($pool . '.yml');
 }
+
+// Yoochoose settings
+if ($value = getenv('YOOCHOOSE_CUSTOMER_ID')) {
+    $container->setParameter('ez_recommendation.default.yoochoose.customer_id', $value);
+}
+
+if ($value = getenv('YOOCHOOSE_LICENSE_KEY')) {
+    $container->setParameter('ez_recommendation.default.yoochoose.license_key', $value);
+}
+
+if ($value = getenv('YOOCHOOSE_SERVER_URI')) {
+    $container->setParameter('ez_recommendation.default.server_uri', $value);
+}
+
+if ($value = getenv('YOOCHOOSE_INCLUDED_CONTENT_TYPES')) {
```

Comment:
I was thinking about that, what about 
Dockerfile:
```
export YOOCHOOSE_INCLUDED_CONTENT_TYPES='article blog_post place'
```

env.php:
```
$container->setParameter('ez_recommendation.default.recommender.included_content_types', explode(' ', $value));
```
?
I didn't work with Docker, so I don't know if that's a good practice.